diff --git a/package/glibc/glibc.mk b/package/glibc/glibc.mk
index cee03a16..75889107 100644
--- a/package/glibc/glibc.mk
+++ b/package/glibc/glibc.mk
@@ -199,6 +199,8 @@ endif

 ifeq ($(BR2_PACKAGE_GLIBC_UTILS),y)
 GLIBC_TARGET_UTILS_USR_BIN = posix/getconf elf/ldd
+GLIBC_TARGET_UTILS_USR_BIN += locale/localedef locale/locale
+GLIBC_TARGET_UTILS_USR_BIN += iconv/iconvconfig iconv/iconv_prog
 GLIBC_TARGET_UTILS_SBIN = elf/ldconfig
 ifeq ($(BR2_SYSTEM_ENABLE_NLS),y)
 GLIBC_TARGET_UTILS_USR_BIN += locale/locale
